client-report/gulpfile.js

```diff
@@ -165,7 +166,7 @@ gulp.task('dist', [
 
 function localUploader(params) {
   params.subdir = params.subdir || ''
-  return gulp.dest(path.join(polisConfig.LOCAL_OUTPUT_PATH, params.subdir))
+  return gulp.dest(path.join(config.get('local_output_path'), params.subdir))
```

Comment:
In the context of a global configuration object, this name could be more descriptive; Perhaps something like `client_build_path`? Open to other suggestions.

I _think_ the same path currently gets used for each of the clients, but may be worth verifying this assumption.